docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,315 @@
 
-    trainer.fit(model)
+.. tip::
 
-Log additional information after the **.fit** loop ends
--------------------------------------------------------
+    Make sure to use your API token in your projects. Read more about how to |token|.
 
-You can log additional metrics, images, model binaries or other things after the ``.fit`` loop is over.
-You just need to specify ``close_after_fit=False`` in the ``NeptuneLogger`` initialization.
+Step 5: Pass NeptuneLogger to the Trainer
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Pass instantiated ``NeptuneLogger`` to the ``pl.Trainer``.
 
 .. code-block:: python3
 
-    neptune_logger = NeptuneLogger(
-        api_key="ANONYMOUS",
-        project_name="shared/pytorch-lightning-integration",
-        close_after_fit=False,
-        ...
-    )
-
-**Log test metrics**
+    trainer = pl.Trainer(max_epochs=PARAMS['max_epochs'],
+                         logger=neptune_logger)
 
-.. code-block:: python3
 
-    trainer.test(model)
+Simply pass ``neptune_logger`` to the ``Trainer``, so that lightning will use this logger. Notice, that ``max_epochs`` is from the ``PARAMS`` dictionary.
 
-**Log additional metrics**
+Step 6: Run experiment
+^^^^^^^^^^^^^^^^^^^^^^
+Fit model to the data.
 
 .. code-block:: python3
 
-    from sklearn.metrics import accuracy_score
-    ...
-    accuracy = accuracy_score(y_true, y_pred)
+    model = LitModel()
 
-    neptune_logger.experiment.log_metric('test_accuracy', accuracy)
+    trainer.fit(model, train_loader)
 
-**Log performance charts**
+At this point you are all set to fit the model. Neptune logger will collect metrics and show them in the UI.
 
-.. code-block:: python3
+Explore Results
+^^^^^^^^^^^^^^^
+You just learned how to start logging PyTorch Lightning experiments to Neptune, by using Neptune logger which is part of the lightning library.
 
-    from scikitplot.metrics import plot_confusion_matrix
-    import matplotlib.pyplot as plt
-    ...
-    fig, ax = plt.subplots(figsize=(16, 12))
-    plot_confusion_matrix(y_true, y_pred, ax=ax)
+Above training is logged to Neptune in near real-time. Click on the link that was outputted to the console or |go-here| to explore an experiment similar to yours. In particular check:
 
-    neptune_logger.experiment.log_image('confusion_matrix', fig)
+#. |metrics|,
+#. |params|,
+#. |hardware|,
+#. |metadata| including git summary info.
 
-**Save checkpoints folder after training**
+.. image:: ../_static/images/integrations/lightning_basic.png
+   :target: ../_static/images/integrations/lightning_basic.png
+   :alt: PyTorchLightning neptune.ai integration
 
-.. code-block:: python3
+Check this experiment |exp-link|.
 
-    model_checkpoint = pl.callbacks.ModelCheckpoint(filepath='my/checkpoints')
+|Run on Colab|
 
-    trainer = Trainer(logger=neptune_logger,
-                      checkpoint_callback=model_checkpoint)
-    trainer.fit(model)
+Advanced options
```

Comment:
The Advanced options are on a TODO list I presume.

Reply:
yes